help_to_heat/locale/cy/LC_MESSAGES/django.po

```diff
@@ -2914,3 +2974,12 @@ msgstr "Darganfyddwch a allwch chi gael cymorth gan eich cyngor lleol drwy'r "
 #: help_to_heat/templates/macros.html:127
 msgid "Household Support Fund"
 msgstr "Gronfa Cymorth i Aelwydydd"
+
+#~ msgid "2 years"
+#~ msgstr "2 flynedd"
+
+#~ msgid "24 hours"
+#~ msgstr "24 awr"
+
+#~ msgid "1 minute"
+#~ msgstr "1 funud"
```

Comment:
Are these unused? The extraction seems to think so. If so, we should remove them.